mobile/src/common/enums/ui/app-button-variant.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum AppButtonVariant {
```

Comment:
If it's used only for button variant definition, then it could be a union type

Reply:
then we won't be able to use it in styles(